docs/headless-cms/docs/06-01-docstopic.md

```diff
@@ -0,0 +1,87 @@
+---
+title: DocsTopic
+type: Custom Resource
+---
+
+The `docstopics.cms.kyma-project.io` CustomResourceDefinition (CRD) is a detailed description of the kind of data and the format used to define an orchestrator that creates Asset CRs for a specific asset type. To get the up-to-date CRD and show the output in the `yaml` format, run this command:
+
+```
+kubectl get crd docstopics.cms.kyma-project.io -o yaml
+```
+
+## Sample custom resource
```

Comment:
please change example to something that makes more sense, slack would be a nice example, 

asyncapi: https://raw.githubusercontent.com/slackapi/slack-api-specs/master/events-api/slack_events_api_async_v1.json
openapi: https://raw.githubusercontent.com/slackapi/slack-api-specs/master/web-api/slack_web_openapi_v2.json
markdown: https://raw.githubusercontent.com/slackapi/slack-api-specs/master/README.md

Reply:
It would be nice to get one concise example that includes all the parameters described.

Reply:
I added the sample CR provided by Michal.